web/react-web/src/Content/Graph/Graph.js

```diff
@@ -144,8 +162,26 @@ class Graph extends Component {
             ...predefinedOptions
         };
 
-        let countries = new Set();
+        if (isXAxisRelative) {
+            // cut data series, so first case is from 100 active cases
+            series = series.map((one) => {
+                if (one.type !== 'prediction') {
+                    for (let i = 0; i < one.values.length; i++) {
+                        if (one.values[i] >= 100) {
+                            if (i > 0) {
+                                one = {...one};
+                                one.values = one.values.slice(i);
+                                one.date_list = one.date_list.slice(i);
+                            }
+                            break;
+                        }
+                    }
```

Comment:
Na toto v Pythone potrebujeme dva riadky, nie sú na to nejaké funkcie v JS?

Reply:
Tak, toto viem dat aj do jedneho, necitatelneho riadku ;). Skratil som to o trochu, no 2 riadky to nie su. JS nema tolko syntaktickeho cukru ako python. Inak, ako by to v pythone vyzeralo?

Reply:
```
start_idx = np.argmax(country_report.cumulative_active >= self.min_case_count)
cumulative_active = country_report.cumulative_active[start_idx:]
```